### PR TITLE
Completed Array-Filter.

### PR DESCRIPTION
**Pull request by stefancarrera**
array-filter

### CODE REVIEW

File: array-filter/main.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  'Janese',
+  'Donna'
+];
+
+function evenNum(num) {
+  for (let i = 0; num >= i; i++) {
+    if ((num % 2 === 0) || (num === 0)) {
+      return true;
+    }
+  }
```
Comment (scbowler) [inline array-filter/main.js:20]: You don't need to use a for loop here, the filter method already loops through the data for you

File: array-filter/main.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+function overFive(num) {
+  for (let i = 5; num > i; i++) {
+    if (num > 5) {
+      return true;
+    }
+  }
```
Comment (scbowler) [inline array-filter/main.js:28]: Same here, you don't need an extra loop

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple updates needed. Let me know if you have any questions

**Comment by stefancarrera** [comment]
Updated the functions to not loop through the arrays

File: array-filter/main.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna'
+];
+
+const overFive = numbers.filter(num => num > 5);
+
+const evenNum = numbers.filter(num => (num % 2 === 0) || (num === 0));
```
Comment (scbowler) [inline array-filter/main.js:17]: You don't really need a special check for `0`. `0 % 2` will equal `0`

**Review by scbowler — APPROVED** [review]
Looks good! 👍 

1 FYI comment, no changes required

**Pull request merged**